applications/nrf_desktop/src/hw_interface/Kconfig.state_switch

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "State switch configuration"
+
+choice
+	prompt "Select state switch interface"
+	default DESKTOP_STATE_SWITCH_NONE
+
+config DESKTOP_STATE_SWITCH_NONE
+	bool "Disable state switch"
+
+config DESKTOP_STATE_SWITCH_ENABLE
+	bool "Enable state switch"
```

Comment:
Could this not just be a single bool config, DESKTOP_STATE_SWITCH.

Why the choice?

Reply:
We will have to add this module based on button events (for other device, which has no hardware switch)

Reply:
I still don't understand, but if the possibility of using a bool has been considered then I trust that it makes sense with a choice.

Reply:
There will be 3rd option here (STATE_SWITCH_BUTTONS), it will be used by other board

Reply:
Ah, I see, thank you